### ISSUE TITLE
Why the swagger does me resolve .ts files?

### ISSUE THREAD

**Issue by FieryCod** [swagger-api/swagger-node#486]
Question same as in title. Can I point in . yaml  controllers to a  `ts` file and in operationId point to a function implemented in a 'ts' file? If so please tell me how to do it.

**Comment by 0v3rst33r** [comment]
@FieryCod Did you come right?  I am also interested in this.

**Comment by meisterlampe** [comment]
Typescript compiles to JavaScript. So it is no problem to use it.
Just change all *.js files to *.ts files and add some types etc. . Compile the project and run it from the output folder. Haven't tried to use classes though.

**Comment by FieryCod** [comment]
@meisterlampe Why compile? There is `ts-node` so he can run `.ts` files via `ts-node`. 
@0v3rst33r We did not choose the `swagger-node` , but [swagger-tools](https://github.com/apigee-127/swagger-tools)

**Comment by meisterlampe** [comment]
@FieryCod Right, but ts-node also transpiles to JavaScript and uses normal node afterwards.

**Comment by 0v3rst33r** [comment]
@FieryCod I had a look at `swagger-tools` now and looks quite handy in terms of a lot of boiler plate code one does not have to do.  I do see they are busy splitting it up in 3 projects ([Deprecating swagger-tools](https://github.com/apigee-127/swagger-tools/issues/335)).

**Comment by FieryCod** [comment]
The maintainer of  `swagger-tools` told me about month ago (June 2017) that in short time there will be the next release of `swagger-tools` so I do not think it gonna be deprecated in couple of years. But of course I am not 100% sure. You can ask maintainer when specifically the new release is likely to happen on apogee's `swagger-tools` gitter.
